tests/assertions/ocp4/rhcos4-moderate-4.14.yml

```diff
@@ -0,0 +1,1447 @@
+rule_results:
```

Comment:
Looks like the `rhcos4-e8` CI is failing on 4.13 because we don't have an assertion file for that scenario, yet:

```
 2024/04/22 16:13:16 E2E-INFO: No global test file or current version test file found, checking for other versioned files in /go/src/github.com/ComplianceAsCode/content/linux_os/guide/auditing/auditd_configure_rules/audit_login_events/audit_rules_login_events_faillock/tests/ocp4
    helpers.go:836: E2E-FAILURE: the rule directory tests/ocp4 contains versioned files, but none for 4.13
```

Reply:
I proposed this in a separate patch so that we don't need to respin this one if we don't have to.

https://github.com/ComplianceAsCode/content/pull/11866